### PR TITLE
Add warnings to how-to-clear documents

### PR DESCRIPTION
**Pull request by matthewrsj**
This project provides very simple tutorials and overviews of methods
that may change. Many users are stumbling on this repository from a
google search and expecting it to 1) be a complete usage documentation
on how to develop a Clear Linux derivative and 2) that the methods
described in this documentation are exhaustive and up-to-date. Add
warnings to redirect users to the officially maintained and centrally
managed documentation we expect them to use.

Signed-off-by: Matthew Johnson <matthew.johnson@intel.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,11 +2,17 @@
...
+
 ## What is this?
 
-This project contains documents that describe how to create a Clear
-Linux OS derivative, which is a Linux OS that uses most, or all, of
-the Clear Linux OS content and organization, but differs in some way.
+This project contains a brief tutorial on how to create a very simple Clear
+Linux OS derivative, which is a Linux OS that uses most, or all, of the Clear
+Linux OS content and organization, but differs in some way. It is intended to
+serve as an example as what can be done with Clear Linux developer tooling. For
```
Comment (kevincwells) [inline README.md]: `s/example as/example of/`

Comment (matthewrsj) [reply README.md]: fixed

**Comment by ahkok** [comment]
Send me fixes to the things that are outdated instead please.

It takes all of 5 minutes to follow through this training and find things that possibly need updating.

References to full tool documentation is entirely reasonable. But not if it includes a liability waiver.

**Comment by matthewrsj** [comment]
> But not if it includes a liability waiver.

Why? The main issue is that these docs will be very *incomplete* regardless of how much work you do on it. The Clear Linux docs team isn't updating this yet people are being told (or otherwise believe) that this is the official documentation on how to run autospec, mixer etc. This is not the official documentation so there needs to at least be a pointer to the official documentation if not a warning that these documents do not cover everything.

**Comment by ahkok** [comment]
There's no need for a warning. Just include references to reference documentation. Everyone will understand what that means. Heck, put it as the first paragraph in each section at the top.

**Comment by matthewrsj** [comment]
Ok, I'll open a new PR

**Pull request closed without merging**